Makefile

```diff
@@ -13,12 +13,12 @@
 # limitations under the License.
 
 IMAGE=amazon/aws-ebs-csi-driver
-VERSION=testing
+VERSION="0.1.0-alpha"
 
 .PHONY: aws-ebs-csi-driver
 aws-ebs-csi-driver:
 	mkdir -p bin
-	go build -o bin/aws-ebs-csi-driver ./cmd/aws-ebs-csi-driver
+	go build -ldflags "-X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.VendorVersion=${VERSION}" -o bin/aws-ebs-csi-driver ./cmd/
```

Comment:
This won't the binary binary that goes into the image. The `Dockerfile` is not calling `make`, it's calling `go build` directly.

We should change the `Dockerfile` to use `make` instead. We need to install it first, if we want to keep using `golang:1.11.1-alpine3.8`.

Reply:
updated to use golang1.11.1-stretch as build image